### DISCUSSION TITLE
integrazione mqtt dopo aggiornamento a 2022.10.5

### DISCUSSION THREAD

**Discussion by sp2003** [indomus/forum#2010] [HUB: Home Assistant]
ciao a tutti

aggiornando HA core da 2022.9.7 a 2022.10.5 non carica più l'integrazione mqtt

se provo a riconfigurarla ottengo l'errore 

`Impossibile caricare il flusso di configurazione: {"message":"Invalid handler specified"}`

tornando indietro alla versione di settembre, riconfigurando mqtt funzia tutto

nel log ho trovato questo

```
2022-10-22 12:45:58.233 ERROR (MainThread) [homeassistant.setup] Setup failed for mqtt: Unable to import component:
IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!
Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.
We have compiled some common reasons and troubleshooting tips at:
https://numpy.org/devdocs/user/troubleshooting-importerror.html
Please note and check the following:
* The Python version is: Python3.9 from "/srv/homeassistant/bin/python3"
* The NumPy version is: "1.23.1"
and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.
Original error was: libcblas.so.3: cannot open shared object file: No such file or directory
```

cercando su google non trovo nulla

è capitato a qualcuno?

**Comment by snakuzzo** [comment]
Ma stai usando core in venv? 
Nel caso, dovresti aggiornare Python alla 3.10 e ricreare il venv

**Comment by sp2003** [comment]
hai ragione, era quello e me ne ero dimenticato....

venv inizia a stancarmi , quasi quasi passo a docker

**Reply by snakuzzo** [reply]
Butta venv

**Reply by KC-inDomus** [reply]
> hai ragione, era quello e me ne ero dimenticato....
> 
> venv inizia a stancarmi , quasi quasi passo a docker

ottima idea
